new_platforms/include/oehost.h

```diff
@@ -24,7 +24,7 @@ extern "C" {
 typedef intptr_t ssize_t;
 #include <openenclave/bits/result.h>
 
-#ifndef __in_ecount
+#ifdef LINUX
```

Comment:
See earlier comment.  Probably
#ifndef _MSC_VER
would be more correct.

Reply:
Same.